object_tracker/match.py

```diff
@@ -70,40 +86,19 @@ def _calc_optical_flow(self, p0, image):
             ).astype("uint8")
         else:
             st = np.squeeze(st1)
-        return st, p1
-
-    def __call__(self, image):
-        image = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
-        p0 = np.squeeze(self.features)
-
-        if len(p0) == 0:
-            self._find_features(image)
-            self._prev_image = image
-            return None
-
-        st, p1 = self._calc_optical_flow(p0, image)
-        self._find_features(image)
-        self._prev_image = image
         if len(np.squeeze(p0[st > 0])) < self._params.min_points_for_find_homography:
```

Comment:
For readability purposes, extract p0[st > 0] and p1[st > 0] to variables.

Reply:
i not think its necessary it is just indexing